### PR TITLE
Updated to version 2.2.0 and Kafka 3.2.0

### PR DESCRIPTION
**Pull request by ppatierno**

### CODE REVIEW

File: operator-metadata/manifests/bundle.clusterserviceversion.yaml [hunk 1]
```diff
@@ -1323,8 +1323,8 @@ spec:
   relatedImages:
```
Comment (kyguy) [inline operator-metadata/manifests/bundle.clusterserviceversion.yaml:1323]: There may be a new openJDK image available but we will have to be careful about updating the version in order the MD5 problem we had last time. We may not need to update anything here. Feel free to save this change for another PR!

**Review by kyguy — COMMENTED** [review]
Lookng good @ppatierno, just a few nits!

File: operator-metadata/manifests/bundle.clusterserviceversion.yaml [hunk 2]
```diff
@@ -792,7 +792,7 @@ spec:
     This release supports only the API version `v1beta2` and CRD version `apiextensions.k8s.io/v1`. 
     If upgrading from AMQ Streams 1.7, migration to `v1beta2` needs to be completed for all AMQ Streams CRDs and CRs before the upgrade to 1.8 is done! 
     If upgrading from AMQ Streams version earlier than 1.7, you need to first install the CRDs from AMQ Streams 1.7 and complete the migration to `v1beta2` for all AMQ Streams CRDs and CRs before the upgrade to 1.8 is done! 
-    For more details about the CRD upgrades, see the [documentation](https://access.redhat.com/documentation/en-us/red_hat_amq_streams/2.1/html/deploying_and_upgrading_amq_streams_on_openshift/assembly-upgrade-str#assembly-upgrade-resources-str).
+    For more details about the CRD upgrades, see the [documentation](https://access.redhat.com/documentation/en-us/red_hat_amq_streams/2.2/html/deploying_and_upgrading_amq_streams_on_openshift/assembly-upgrade-str#assembly-upgrade-resources-str).
```
[the remark this answers is not in the document]
Comment (ppatierno) [reply operator-metadata/manifests/bundle.clusterserviceversion.yaml:795]: I just updated the version here as well. Of course the path could be completely different but it's not a blocker for now imho.

File: operator-metadata/manifests/bundle.clusterserviceversion.yaml [hunk 3]
```diff
@@ -818,7 +818,7 @@ spec:
     
     The AMQ Streams Operator understands how to run and upgrade between a set of Kafka versions.
     When specifying a new version in your config, check to make sure you aren't using any features that may have been removed.
-    See [the upgrade guide](https://access.redhat.com/documentation/en-us/red_hat_amq_streams/2.1/html/deploying_and
+    See [the upgrade guide](https://access.redhat.com/documentation/en-us/red_hat_amq_streams/2.2/html/deploying_and
```
[the remark this answers is not in the document]
Comment (ppatierno) [reply operator-metadata/manifests/bundle.clusterserviceversion.yaml:821]: Ditto as above

File: operator-metadata/manifests/bundle.clusterserviceversion.yaml [hunk 4]
```diff
@@ -1114,9 +1114,9 @@ spec:
...
                 com.company: Red_Hat
                 rht.prod_name: Red_Hat_Integration
-                rht.prod_ver: 2022.Q2
+                rht.prod_ver: 2022.Q3
                 rht.comp: AMQ_Streams
-                rht.comp_ver: "2.1"
+                rht.comp_ver: "2.2"
                 rht.subcomp: cluster-operator
                 rht.subcomp_t: infrastructure
             spec:
```
[the remark this answers is not in the document]
Comment (ppatierno) [reply operator-metadata/manifests/bundle.clusterserviceversion.yaml:1122]: good catch! Thanks!

File: operator-metadata/manifests/bundle.clusterserviceversion.yaml (hunk 1 above)
Comment (ppatierno) [reply operator-metadata/manifests/bundle.clusterserviceversion.yaml:1323]: wdym? We are going anyway to remove Kafka 3.0.0 to be replaced by 3.2.0. The SHAs are not the final ones of course.

Comment (kyguy) [reply operator-metadata/manifests/bundle.clusterserviceversion.yaml:1323]: Talking about the strimzi-maven-builder image int the relatedImages section but yes let's save the change for when we update the other MD5s

**Pull request merged**